lib/ui/pages/account_dashboard.dart

```diff
@@ -0,0 +1,351 @@
+                                    fontSize: 28,
+                                  ),
+                                ),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+                const Divider(
+                  height: 8,
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                    20.0,
+                    0,
+                    20,
+                    0,
+                  ),
+                  child: GetBuilder<AccountDashboardController>(
+                    builder: (_) {
+                      return ListTile(
+                        contentPadding: const EdgeInsets.symmetric(),
+                        title: const TitleText(
+                          text: 'From',
+                          fontSize: 14,
+                        ),
+                        subtitle: Text(
+                            Utils.getSecretAccountNumber(_.selectedAccount)),
+                        trailing: TitleText(
+                          text: _.selectedAccount.name,
+                          fontSize: 14,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                    20.0,
+                    0,
+                    20,
+                    0,
+                  ),
+                  child: ListTile(
+                    contentPadding: const EdgeInsets.symmetric(),
+                    title: const TitleText(
+                      text: 'To',
+                      fontSize: 14,
+                    ),
+                    subtitle: Text(
+                      Utils.getSecretAccountNumberFromString(t.to),
+                    ),
+                    trailing: TitleText(
+                      text: t.payeeName,
+                      fontSize: 14,
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                    20.0,
+                    0,
+                    20,
+                    0,
+                  ),
+                  child: ListTile(
+                    contentPadding: const EdgeInsets.symmetric(),
+                    title: const TitleText(
+                      text: 'Date',
+                      fontSize: 14,
+                    ),
+                    // subtitle: Text(selectedAccount.accountNumber),
+                    trailing: TitleText(
+                      text: t.date,
+                      fontSize: 14,
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    ));
+  }
+
+  Widget _transaction(Transaction t) {
```

Comment:
perhaps you can create a widget for this instead of defining it as part of the account_dashboard class?

Reply:
But it's specific to this class. We aren't using this anywhere else. I am putting generic widgets in the 'widgets' folder. Does that make sense ?

Reply:
I think `widgets` folder does not necessarily contain a very generic widget. You can create a folder for those who are more specific. I just feel that it will avoid mixing the business logic and ui display for account dashboard.

Reply:
This will also allow us to follow the SR and SoC principles :)

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12